homeassistant/components/lcn/manifest.json

```diff
@@ -8,5 +8,5 @@
   "documentation": "https://www.home-assistant.io/integrations/lcn",
   "iot_class": "local_push",
   "loggers": ["pypck"],
-  "requirements": ["pypck==0.7.21", "lcn-frontend==0.1.6"]
+  "requirements": ["pypck==0.7.22", "lcn-frontend==0.1.6"]
```

Comment:
I was about to say, please split off, and then I remembered there already was a PR for this